component-tools/src/main/java/org/talend/sdk/component/tools/CarBundler.java

```diff
@@ -69,6 +69,7 @@ public void run() {
         final Properties metadata = new Properties();
         metadata.put("date", date);
         metadata.put("version", ofNullable(configuration.version).orElse("NC"));
+        metadata.put("CarMainVersion", Metadata.get().getVersion());
```

Comment:
If you want to embed component-runtime build plugin version, the preferred way is using java-template.

how-to sample:
- Declaration: https://github.com/Talend/component-runtime/blob/9fb59c061ff3c073fc7655a184b1e73c0d209fd0/component-runtime-beam/src/main/java-templates/org/talend/sdk/component/runtime/beam/Versions.java#L24
- Usage: https://github.com/Talend/component-runtime/blob/c4dd3cca4784267ca9892ed01a868f022907111f/component-runtime-beam/src/main/java/org/talend/sdk/component/runtime/beam/spi/BeamComponentExtension.java#L136

then 
`metadata.put("CarMainVersion",  Versions.VERSION);`
Also, I suggest renaming `CarMainVersion` to `CarBundlerVersion`.

Reply:
Got it, I'll update the codes, thank you very much!